runtime/common/src/main/proto/ControlMessage.proto

```diff
@@ -117,8 +117,6 @@ message BlockStateChangedMsg {
 message DataSizeMetricMsg {
     // TODO #96: Modularize DataSkewPolicy to use MetricVertex and BarrierVertex.
     repeated PartitionSizeEntry partitionSize = 1;
-    required string blockId = 2;
```

Comment:
id of the metric vertex?
The runtime master needs to know which metric vertex is this proto for?

Reply:
Now metric vertex only acts as a barrier, and all run-time optimization is done via ExecutionProperties of IREdges. So id of metric vertex isn't needed anymore.